docs/1-Applying/resumes.md

```diff
@@ -104,17 +116,16 @@ Consistency:
 - use periods at the end of bullet points or don't, but don't mix them
 - make sure your dates are in the same format (e.g. Aug 2017 or September '17)
 - in general, the way you format each section and the way you talk about things should be consistent - this makes it easy to find information and to scan your resume
-- in case you're thinking this right now - yes I could have better consistency in this guide because I'm not alawys doing the things I literally just mentioned ...but it's a lot harder for a thing this big and I guess I don't care enough ^_^
+- in case you're thinking this right now - yes I could have better consistency in this guide because I'm not alawys doing the things I literally just mentioned ...but it's a lot harder for a thing this big and I guess I don't care enough ^.^
```

Comment:
The syntax highlighter for Markdown in vim was getting confused by the _. It thought that the entire rest of the file was supposed to be highlighted... ^^'